frontend/src/metabase/nav/components/ProfileLink/ProfileLink.jsx

```diff
@@ -157,6 +164,10 @@ function ProfileLink({ adminItems, onLogout }) {
           )}
         </Modal>
       ) : null}
+
+      {modalOpen === "diagnostic" && (
+        <ErrorDiagnosticModalWrapper isModalOpen={true} onClose={closeModal} />
```

Comment:
Something else is that this modal already exists in the React Component tree in `App.tsx` as `<KeyboardTriggeredErrorModal />`. Rather than have it here twice, can the "Filke a bug" action just dispatch `openDiagnostics()` as well? Rather than have the modal in the component tree twice?